src/main.ts

```diff
@@ -37,7 +37,7 @@ export const vibClient = clients.newClient({
   timeout: 10000,
   headers: {
     "Content-Type": "application/json",
-    "User-Agent": `vib-action/${process.env.GITHUB_ACTION_REF}`,
+    "User-Agent": `vib-action/${userAgentVersion}`,
```

Comment:
Do you mean `version`? This is not the user-agent version, but the version of the GitHub Action.

Reply:
This change makes sense to me. But, do you want the same version for all releases?

Reply:
I don't. We need to identify the GH version through the user-agent header.

Reply:
ok, so in this case it should be ok changing ``vib-action/${userAgentVersion}`` with `via-action/${ghActionVersion}` and also the constant, right?